book/src/chapters/quick_start.md

```diff
@@ -0,0 +1,94 @@
+# Quick Start
+
+**Get ros-z running in under 5 minutes with this hands-on tutorial.** Build a complete publisher-subscriber system to understand the core concepts through working code.
+
+```admonish tip
+This guide assumes basic Rust knowledge. If you're new to Rust, complete the [Rust Book](https://doc.rust-lang.org/book/) first for the best experience.
+```
+
+## Setup
+
+Add ros-z dependencies to your `Cargo.toml`:
+
+```toml
+[dependencies]
+ros-z = "*"
+ros-z-msgs = "*"  # Standard ROS 2 message types
+tokio = { version = "1", features = ["full"] }  # Async runtime
+```
+
+```admonish note
+An async runtime is required for ros-z. This example uses Tokio, the most popular choice in the Rust ecosystem.
+```
+
+## Your 1st Example
+
+Here's a complete publisher and subscriber in one application:
+
+```rust,ignore
+{{#include ../../../ros-z/examples/z_pubsub.rs}}
```

Comment:
Adding some comments in the examples would help the reader to understand the different parts of the code, moreover for those not familiar with Rust.